### PR TITLE
Util shortcuts

### PR DESCRIPTION
**Pull request by LeonardoZivieri**
New features on double and right click
Every buttons has shortcuts and show the keys on hover

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -51,6 +51,14 @@ $(document).ready(() => {
...
         $(".led").click(onLedClicked);
         $(".led").dblclick(() => $("#color").change());
+        $(".led").contextmenu(function (e) {
+            $(this).click();
+            e.preventDefault();
+            // Maybe is not the best
+            let last_color = $("#color").val();
+            $("#color").val('000000').change();
+            $("#color").val(last_color);
+        });
```
Comment (nicolasdeory) [inline app.js]: Really like the changes. Perhaps reset frame should be a hotkey like CTRL+R or backspace? Right-click can be triggered accidentally and could be annoying especially with no undo functionality

Comment (LeonardoZivieri) [reply app.js]: I can make another shortcut, here is a list of possible shortcuts that may be better than the right click:

 - Ctrl + R
 - Backspace
 - Space
 - Long Click
 - Ctrl + Click

Which is the best?

**Comment by LeonardoZivieri** [comment]
I used the Ctrl + Click option to facilitate the creation of a complete frame with ctrl + mouse easily

**Comment by LeonardoZivieri** [comment]
I used the Ctrl + Click option to facilitate the creation of a complete frame with ctrl + mouse easily

File: app.js (hunk 1 above)
Comment (nicolasdeory) [reply app.js]: I thought that you meant that when you pressed right-click **all** of the LEDs would go black. If it's just one led, I'd say right-click is fine.

**Comment by nicolasdeory** [comment]
Change the reset LED shortcut from Ctrl+Click to right-click whenever you can. Everything else looks good 👍

**Pull request merged**